--- a/src/Makefile.test.include
+++ b/src/Makefile.test.include
@@ -69,12 +69,14 @@ RAVEN_TESTS =\
   test/pmt_tests.cpp \
   test/pqkey_tests.cpp \
   test/pqkey_hardening_tests.cpp \
+  test/rip25_versionbits_tests.cpp \
   test/policyestimator_tests.cpp \
   test/pow_tests.cpp \
   test/prevector_tests.cpp \
   test/kawpow_tests.cpp \
+  test/kawpow_v48_hardening_tests.cpp \
   test/raii_event_tests.cpp \
   test/random_tests.cpp \
   test/reverselock_tests.cpp \
